pkg/network/listener.go

```diff
@@ -270,18 +275,28 @@ func (l *listener) Close(lctx context.Context) error {
 
 func (l *listener) listen(lctx context.Context) error {
 	var err error
-	var rawl *net.TCPListener
+	var rawl net.Listener
 	var rconn net.PacketConn
 
+	if l.localAddress == nil {
+		return errors.New("listener local addr is nil")
+	}
+
 	switch l.network {
 	case "udp":
 		lc := net.ListenConfig{}
 		if rconn, err = lc.ListenPacket(context.Background(), l.network, l.localAddress.String()); err != nil {
 			return err
 		}
 		l.packetConn = rconn
+	case "unix":
```

Comment:
Need to add a description.
BTW If the UDS is already listened by another MOSN instance, it will be a problem to delete it directly.